web/src/state/index.ts

```diff
@@ -97,6 +98,29 @@ export type Workspace = {
 const workspace = atomWithStorage<Workspace | undefined>("workspace", undefined);
 export const useWorkspace = () => useAtom(workspace);
 
+const sessionStorageObj: SyncStorage<Workspace | undefined> = {
+  getItem: key => {
+    const value = sessionStorage.getItem(key);
+    if (value === null) {
+      return undefined;
+    }
+    return JSON.parse(value) as Workspace;
+  },
+  setItem: (key, value) => {
+    sessionStorage.setItem(key, JSON.stringify(value));
+  },
+  removeItem: function (): void {
+    throw new Error("Function not implemented.");
```

Comment:
If we log-outed application or the member is removed from workspace, it might cause something error:eyes:

Reply:
if the user log-out the saved session won't be removed , but if the user log-in again and he was the same one logged last time system will retrieve the last workspace he was in and if a new user logged -in then system will take him to the default work space

Reply:
@keiya01 kindly note that I tested above scenario and it is working fine